pkg/kubectl/cmd/apply_test.go

```diff
@@ -990,6 +997,97 @@ func TestRunApplySetLastApplied(t *testing.T) {
 	cmdutil.BehaviorOnFatal(func(str string, code int) {})
 }
 
+func TestRunApplyDiffLastApplied(t *testing.T) {
```

Comment:
I think it will make more sense to move `diff_last_applied`'s test to its own file like what we do for create subcommands tests. 
I'm fine to resolve it in a followup PR. `set_last_applied` is already in this file. And you will also move these to `apply`'s own pkg.